src/Http/Http.Results/src/ResultsOfT.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.AspNetCore.Http.HttpResults;
 /// <remarks>
 /// An instance of this type cannot be created explicitly. Use the implicit cast operators to create an instance
 /// from an instance of one of the declared type arguments, e.g.
-/// <code>Results&lt;OkObjectHttpResult, ProblemHttpResult&gt; result = Results.Ok();</code>
```

Comment:
Can we rename this file to `ResultsOfT.Generated.cs` similar to how we did for `ResultsOfTTests.Generated.cs`?